index.bs

```diff
@@ -7052,6 +7052,14 @@ the [=[RP]=] could mitigate the privacy leak using the same approach of returnin
 as discussed in [[#sctn-username-enumeration]].
 
 
+# Accessibility Considerations # {#sctn-accessiblility-considerations}
+
+[=Client platforms=], at [=registration=] time, should provide affordances for users to complete [=authorization gestures=] correctly. This could involve naming the authenticator, choosing a picture to associate with the device, or entering freeform text instructions.
+
+[=Ceremonies=] relying on timing, e.g., during registration (see {{PublicKeyCredentialCreationOptions/timeout}}), ought to follow [WCAG Guideline 2.2 Enough Time](https://www.w3.org/WAI/WCAG21/Understanding/enough-time).
```

Comment:
The API allows specifying a timeout. If the RP-provided timeout is in conflict with this line (e.g. something very short like 5 seconds), should the user agent be empowered to extend the ceremony timeout?

Reply:
I think so (?) given [WCAG Guideline 2.2 Enough Time] ?  Tho would be good for others with more expertise in this particular aspect of W3C specs to weigh-in ?

Reply:
Note that, in practice, browsers already bound the RP-provided timeout to sane values.

Reply:
( doh, shudda looked at the code... )

So I interpret that as: adding that 2nd sentence is OK

Reply:
done in 00a5760

Reply:
Yes, thanks for adding that additional sentence/clarification.